subsys/net/ip/Kconfig

```diff
@@ -472,6 +472,19 @@ config NET_BUF_DATA_POOL_SIZE
 	 This value tell what is the size of the memory pool where each
 	 network buffer are allocated from.
 
+config NET_HEADERS_ALWAYS_CONTIGUOUS
```

Comment:
Offering this kind of option is imho dangerous, someone will misuse it anyway. How much memory we are saving with this, what does ram_report say if we enable/disable it?

Reply:
for udp/ipv4 setup, even on 128bytes frag size this will be just fine option (it is dangerous at the same level as wrongly teawking frag size: user has to know what he is doing). It saves stack actually.

Reply:
Sure, it probably save something. Just wondering do you have numbers about the savings.